Enable/Utilities/Utilities.m

```diff
@@ -242,12 +250,25 @@ + (void) getLocationsFromLocation: (CLLocationCoordinate2D) location corner: (CL
         }
     }];
 }
-
++ (bool) shouldUpdateLocation: (GMSProjection * _Nonnull) prevProjection currentRegion: (GMSVisibleRegion) currentRegion radius: (double) radius prevRadius: (double) prevRadius {
+    if([prevProjection containsCoordinate: currentRegion.farRight] && [prevProjection containsCoordinate: currentRegion.farLeft] && [prevProjection containsCoordinate: currentRegion.nearRight] && [prevProjection containsCoordinate: currentRegion.nearLeft]){
+        if(radius > kMaxRadius) {
+            return false;
```

Comment:
Same thing here, shouldn't you return false only when prevRadius was already equal to kMaxRadius?
I'm thinking, if prevRadius is let's say 32, and radius is 60, you don't want to return false, you want to return true, and just set radius to 50, no?